qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
If you know which tests, will not be easier for you to point me to
them vs. telling me to look at all of them? Please be constructive.

Note that minimizing the number of variables makes sense, will fix,
but these templates have almost not variables so complexity is low.